articles/intro-to-katas.md

```diff
@@ -27,47 +28,59 @@ You can use the Katas for learning in your environment of choice:
+Explore the basics and fundamentals of quantum computing, dive deeper into quantum algorithms and protocols, and solve some quantum based games. We recommend you to follow this learning path in the beginning to make sure you have a solid grasp on the fundamental concepts of quantum computing. Of course, you can skip the topics you're comfortable with, such as complex arithmetic, and learn the algorithms in any order you want.
 
 ### Introduction to quantum computing concepts
 
-* [Complex arithmetic](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/ComplexArithmetic)
-* [Linear algebra](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/LinearAlgebra)
-* [The concept of a qubit](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/Qubit)
-* [Single-qubit quantum gates](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/SingleQubitGates)
-* [Multi-qubit systems](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/MultiQubitSystems)
-* [Multi-qubit gates](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/MultiQubitGates)
+| Kata | Description |
+|:-----|-------------|
+|[Complex arithmetic](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/ComplexArithmetic)|This tutorial explains some of the mathematical background required to work with quantum computing, such as imaginary and complex numbers.|
+|[Linear algebra](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/LinearAlgebra)|Linear algebra is used to represent quantum states and operations in quantum computing. This tutorial covers the basics, including matrices and vectors.|
+|[The concept of a qubit](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/Qubit)|Learn about qubits - one of the core concepts of quantum computing. |
+|[Single-qubit quantum gates](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/SingleQubitGates)|This tutorial introduces single-qubit quantum gates, which act as the building blocks of quantum algorithms and transform quantum qubit states in various ways.|
+|[Multi-qubit systems](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/MultiQubitSystems)|This kata introduces multi-qubit systems, their representation in mathematical notation and in Q# code, and the concept of entanglement.|
+|[Multi-qubit quantum gates](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/MultiQubitGates)|This tutorial follows up the [Single-qubit quantum gates](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/SingleQubitGates) kata, and focuses on applying quantum gates to multi-qubit systems.|
```

Comment:
The distinction between a kata and a tutorial can be a bit hazy, the rule of thumb is: if it has theory in it and is available only as a Jupyter Notebook, it's a tutorial, if it only has exercises and is also available as a VS/VS Code project, it's a kata :-) 
```suggestion
|[Multi-qubit quantum gates](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/MultiQubitGates)|This tutorial follows the [Single-qubit quantum gates](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/SingleQubitGates) tutorial, and focuses on applying quantum gates to multi-qubit systems.|
```